src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{" +
+                "color=" + this.getColor() +
+                ", number=" + this.getNumber() +
+                '}';
```

Comment:
In the toString() method, you should use direct field access instead of calling getters. This is because the toString() method is intended to provide a string representation of the object's state, and the state is represented by the fields themselves. Using direct field access is more conventional in this case.